flaml/tune/tune.py

```diff
@@ -148,15 +147,16 @@ def compute_with_config(config):
 
     Args:
         training_function: A user-defined training function. 
-        init_config: A dictionary from a subset of controlled dimensions
-            to the initial low-cost values. e.g.,            
+        config: A dictionary to specify the search space.
+        points_to_evaluate: Initial parameter suggestions to be run first. 

```

Comment:
change to 'points_to_evaluate: A list of initial hyperparameter configuration suggestions to be run first.' ?

Reply:
done